### PR TITLE
home page

### PR DESCRIPTION
**Pull request by thaotonto**

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<footer>
+  <small><%= t(".credit_html") %></small>
```
Comment (ducbet) [inline app/views/layouts/_footer.html.erb]: ()

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,15 +1,17 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>MovieNerd</title>
+    <title><%= full_title(yield(:title)) %></title>
```
Comment (ducbet) [inline app/views/layouts/application.html.erb]: full_title yield(:title)

**Comment by ducbet** [comment]
ok

**Comment by noobsi** [comment]
ok

**Pull request merged**